src/main/java/com/google/codeu/servlets/ImagesServlet.java

```diff
@@ -0,0 +1,24 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Handles fetching all users for the community page.
+ */
+
+ @WebServlet("/images")
+ public class ImagesServlet extends HttpServlet {
+
+   @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
```

Comment:
Have the @override align with the line below, it looks like there may be an extra space